go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/kubeflow/pipelines
 
 require (
+	cloud.google.com/go v0.88.0 // indirect
```

Comment:
whenever you update dependencies, you need to update licenses
https://github.com/kubeflow/pipelines/tree/master/backend#updating-licenses-info

Reply:
Thank you Yuan for the reference!

I have downloaded the go-licenses tool and ran the following command, which generates the license_info.csv file as included in the PR. It is my first time using the tool, can you help me with whether I am doing it correctly?

Command: `go-licenses  csv ./test/tools/project-cleaner/project-cleaner | tee ./test/tools/project-cleaner/license_info.csv`

Output:

```

"Done: found dependencies" count=14
# Generated by https://github.com/google/go-licenses/v2. DO NOT EDIT.
"Failed" err="licenses not found" module="github.com/kubeflow/pipelines"
"Failed" err="override version mismatch: found v0.88.0, but override is for v0.55.0" module="cloud.google.com/go"
"Failed" err="licenses not found" module="github.com/golang/groupcache"
"Failed" err="licenses not found" module="github.com/golang/protobuf"
"Failed" err="override version mismatch: found v0.23.0, but override is for v0.22.5" module="go.opencensus.io"
github.com/googleapis/gax-go/v2, https://github.com/googleapis/gax-go/blob/v2.0.5/LICENSE, BSD-3-Clause
"Failed" err="licenses not found" module="golang.org/x/net"
"Failed" err="licenses not found" module="golang.org/x/oauth2"
"Failed" err="licenses not found" module="golang.org/x/text"
"Failed" err="licenses not found" module="google.golang.org/api"
"Failed" err="licenses not found" module="google.golang.org/genproto"
"Failed" err="licenses not found" module="google.golang.org/grpc"
google.golang.org/protobuf, https://github.com/protocolbuffers/protobuf-go/blob/v1.27.1/LICENSE, BSD-3-Clause
Failed to scan licenses for 11 module(s)
gopkg.in/yaml.v2, https://github.com/go-yaml/yaml/blob/v2.4.0/LICENSE, Apache-2.0 / MIT
```

Reply:
Hi @zijianjoy, let's discuss this over GVC.
Basically, because go.mod is shared across the entire repo, you need to update licenses for other images (ml-pipeline, ml-pipeline-persistence-agent, ...).

For project cleaner, you don't need to run the tool, because we don't release project cleaner.

Reply:
Sounds good, I will remove the license csv on project cleaner now. 

I tried to run the backend `make all` rule with error:
```
Failed to scan licenses for 104 module(s)
make: *** [license_apiserver] Error 1
```

I guess it is more about how I should follow this example to fix the license issue: https://github.com/Bobgy/go-licenses/blob/main/v2/go-licenses.yaml

Reply:
Discussed offline, updated license for review. Currently commented the override of cloud.google.com/go